network/packet.go

```diff
@@ -101,7 +101,7 @@ func parseNetwork(netdata packet.NetworkData, nameservers []net.IP) ([]Interface
 	for _, iface := range netdata.Interfaces {
 		if iface.Name != "chassis0" && iface.Name != "ipmi0" {
 			bond.slaves = append(bond.slaves, iface.Name)
-			if iface.Name == "enp1s0f0" {
+			if iface.Name == "enp1s0f0" || iface.Name == "ens20" {
```

Comment:
We're going to run through quite a few of the systemd-udev naming conventions, but they all appear to order alphabetically, so I think we can remove this check and go alphabetical.